nodes/SetComp.py

```diff
@@ -1 +1,30 @@
+"""
+SetComp astroid node
+
+This node represents set comprehension expressions in Python, which take the
+form {element for increment in iterable}. This node also makes use of the
+Comprehension node.
+
+Attributes:
+    - elt         (Expr)
+        - The part to be evaluated to make every item in the set. This can
+          be any expression node, such as a BinOp or Str.
+    - generators  (List[])
```

Comment:
In general for the `List` type, either you don't specify the type of the contents and just write `List`, or you *do* specify the type of the contents, e.g. `List[Comprehension]`.